docs/manual/about/under-the-hood/guarantees.md

```diff
@@ -20,15 +20,22 @@ support specific guarantees.
 
 ### At-Least-Once
 
-The `at-least-once` delivery guarantee ensures that an [event][docs.data-model]
-received by a Vector component will be delivered at least once. While rare, it
-is possible for an event to be delivered more than once. See the
-[Does Vector support exactly once delivery](#does-vector-support-exactly-once-delivery)
-FAQ below).
+The `at-least-once` delivery guarantee ensures that an
```

Comment:
`at_least_once` (to be consistent with how it will appear in configs)

Reply:
Fixed (I think)

Reply:
Oh sorry, I meant just in the code word, not in the headings